recipes/IWSLT22_lowresource/README.md

```diff
@@ -0,0 +1,57 @@
+# IWSLT 2022 Low-resource Task: Tamasheq-French end-to-end Speech Translation
+
+
+## Description
+
+This is the recipe for the best system from the IWSLT 2022 low-resource task, as described in the original paper.
+The speech translation model comprises a wav2vec 2.0 encoder and a Transformer decoder. It is trained end-to-end without any auxiliary loss. The recipe allows for removing the last layers of the Transformer Encoder inside the wav2vec 2.0 in order to reduce the number of training parameters.
+
+## Data Pre-processing
+
+For preparing the dataset, please run this command:
+
+```
+bash data_proc/prepare_iwslt22.sh
```

Comment:
Thanks for the changes @mzboito but I will request some more because I am pretty "chiant" :p The only issue remaining is the data preparation. We do not have any shell script, hence, this part should disappear. You can put the downloading (git clone) instruction directly on the README file on a proper section (#Data access or whatever). Then, we do not prepare the data by calling an external script, but rather, directly in the main function (see bellow comment).